### PR TITLE
Load the initial set of block patterns in Core

### PR DESCRIPTION
**Pull request by youknowriad**
Trac ticket: https://core.trac.wordpress.org/ticket/50550

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-includes/block-patterns.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+/**
+ * Loads a Core block pattern.
+ *
+ * @since 5.5.0
+ * @access private
+ *
+ * @param string $name Block pattern name.
+ * @return array Block pattern settings.
+ */
+function _load_block_pattern( $name ) {
```
Comment (mcsf) [inline src/wp-includes/block-patterns.php:18]: Should we minimise the risk of naming collision by inlining the `require` calls?

Comment (youknowriad) [reply src/wp-includes/block-patterns.php:18]: Can you clarify what naming collision you're thinking of?

Comment (mcsf) [reply src/wp-includes/block-patterns.php:18]: Not a big deal, but it's noticeable how such a local helper function ends up polluting the global function namespace.

I guess you could also use an anonymous function ([PHP 5.3](https://www.php.net/manual/en/functions.anonymous.php)), or avoid functions entirely with a for loop looping over the pattern slugs.

Comment (youknowriad) [reply src/wp-includes/block-patterns.php:18]: I like the loop proposal, I'll update the code.

Comment (youknowriad) [reply src/wp-includes/block-patterns.php]: Updated.

Comment (mcsf) [reply src/wp-includes/block-patterns.php]: Cool. Commit! 🚀

**Pull request closed without merging**